pandas/core/groupby/__init__.py

```diff
@@ -0,0 +1,9 @@
+import warnings
+
+# TODO: Remove after 0.XX.x
+warnings.warn("'pandas.core.groupby' has moved. Use "
+              "'pandas.core.groupby.groupby' instead", FutureWarning,
+              stacklevel=2)
+
+from pandas.core.groupby.groupby import (Grouper, GroupBy,  # noqa: F401
```

Comment:
GroupBy, SeriesGroupBy and DataFrameGroupBy were all added here as they were causing downstream failures for Dask, even though they aren't explicitly referenced internally. Once we settle on a go-forward here we'll have to follow up with the Dask team to let them know of impacts

Reply:
yes this is fine. These are basically downstream dev dependencies (e.g. you may want to do something based on the object if its a SeriesGroupBy or a DataFrameGroupBy) so ok with this